test/fixtures/tree/footnote-trailing-whitespace.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
`footnotes` aren’t on by default, so this is only testing non-footnote functionality.

I suggest duplicating this fixture output, adding `.footnotes` to also test footnotes, and then with `node script/regenerate-fixtures` you can check if it all looks fine.

Reply:
I'm not sure what `.footnotes` is and where it's supposed to live.

Reply:
in the filename, somewhere at the end! Per input file, there are several output fixtures. Each individual fixture has different options, as defined in its filename.

Reply:
I hadn't noticed. Pretty cool though.